pkg/standalone/standalone.go

```diff
@@ -1047,10 +1049,27 @@ func downloadFile(dir string, url string) (string, error) {
 		return "", nil
 	}
 
-	resp, err := http.Get(url)
+	client := http.Client{
+		Timeout: 0,
```

Comment:
@hueifeng Why the `Timeout` here is set to 0. If this is required then should not we have some +ve value?

Reply:
Because file can take a while to download on slow connections, so set `Timeout` to 0.

Reply:
@hueifeng This can cause a process to hang forever .... Based on the comments in #962 can a method be added to track incremental progress?

Reply:
There is one suggestion mentioned in the attached issue to check if some data got transferred in last 5 sec or timeout if not.

Reply:
@ItalyPaleAle I add a link to mark, is it okay?